work-process/scrum.md

```diff
@@ -45,6 +45,10 @@ Every day on stand-up scrum master should screen share and go through all the us
 
 Developers should be more involved in marketing/support related user stories. When a user story is created the author should assign a developer (one or more) from whom he/she thinks he/she will be able to get the technical details about the user story. Then that developer (one or more) should update the detailed technical description of the user story. Developers should also try to find different, easier solutions for the problem that this user story will solve, and present them to the author of the user story.
 
+When you put your User Story into review, there is no need to assign the Scrum Master to the story -- they will see it on the Kanban board in the `Review` column.
```

Comment:
Change Kanban for Zenhub board

Reply:
Even ZenHub calls that "the Kanban board" should we really invent a new phrase?

Reply:
After looking through the handbook, perhaps I only considered this making change, not implemented yet. My thought process is that we should definitely not use Kanban as it's different to our board: https://realtimeboard.com/blog/scrum-kanban-boards-differences/

You are right we should not invert a new name but ZenHub boards are inspired-by not actually Kanban boards so this is more correct and by mentioning ZenHub we give the reader a reference point to know what we are using and where to look. 

I did consider just 'board' but that might be too generic, unless ZenHub has already been mentioned in the paragraph.